packages/web/src/components/host-remix-contest-modal/HostRemixContestModal.tsx

```diff
@@ -149,7 +130,7 @@ export const HostRemixContestModal = () => {
               onChange={(value) =>
                 handleChange(value, timeValue, meridianValue)
               }
-              value={contestEndDate}
+              value={contestEndDate?.toISOString() ?? undefined}
```

Comment:
nit: if undefined is acceptable maybe we don't need the `?? undefined`

Reply:
it might have been a null vs undefined thing, but i'll check